src/components/Reports/MinisterHousingAllowance/NewRequest/NewRequestPage.tsx

```diff
@@ -1,17 +1,39 @@
 import { useState } from 'react';
 import { Container, Stack } from '@mui/material';
+import { Formik } from 'formik';
 import { useTranslation } from 'react-i18next';
+import * as yup from 'yup';
+import i18n from 'src/lib/i18n';
 import { useNewStepList } from '../../../../hooks/useNewStepList';
 import { mainContentWidth } from '../MinisterHousingAllowance';
 import { PanelLayout } from '../PanelLayout/PanelLayout';
 import { mocks } from '../Shared/mockData';
-import { NewRequestStepsEnum, PanelTypeEnum } from '../Shared/sharedTypes';
+import {
+  NewRequestStepsEnum,
+  PanelTypeEnum,
+  RentOwnEnum,
+} from '../Shared/sharedTypes';
+import { Receipt } from './Steps/StepFour/Receipt';
 import { AboutForm } from './Steps/StepOne/AboutForm';
 import { Calculation } from './Steps/StepThree/Calculation';
 import { RentOwn } from './Steps/StepTwo/RentOwn';
 import { StepsList } from './StepsList/StepsList';
 
-export const NewRequestPage: React.FC = () => {
+interface NewRequestPageProps {
+  onOpen?: () => void;
+}
+
+export interface FormValues {
+  rentOrOwn: RentOwnEnum | undefined;
+}
+
+const validationSchema = yup.object({
+  rentOrOwn: yup
+    .string()
+    .required(i18n.t('Please select one of the options above to continue.')),
+});
+
```

Comment:
Can we do what we did on the Goal Calculator, where we passed in the schema into the input.
I know this means making our version of the `AutosaveTextField`.